tomy_400.nv.json

```diff
@@ -0,0 +1,112 @@
+{
+	"_notes": "Compiled by Horsepin.",
+	"_copyright": "Copyright (C) 2016 by Horsepin>",
+	"_license": "GNU Lesser General Public License v3.0",
+	"_endian": "big",
+	"_roms": [ "tomy_400" ],
+	"_fileformat": 0.1,
+	"_version": 0.1,
+
+	"last_game": [
+		{
+			"start": "0x05B2",
+			"encoding": "bcd",
+			"length": 5
+		},
+		{
+			"start": "0x05BE",
+			"encoding": "bcd",
+			"length": 5
+		},
+		{
+			"start": "0x05C3",
+			"encoding": "bcd",
+			"length": 5
+		},
+		{
+			"start": "0x05C8",
+			"encoding": "bcd",
+			"length": 5
+		}
+	],
+	"high_scores": [
+		{
+			"label": "Grand Champion",
+			"short_label": "GC",
+			"initials": {
+				"start": "0x16BA",
+				"encoding": "ch",
+				"length": 3
+			},
+			"score": {
+				"encoding": "bcd",
+				"offsets": [ "0x1694", "0x167C", "0x167D", "0x167E", "0x167F" ]
```

Comment:
This looks curious.  Did they have to pull in an extra byte to allow for scores beyond 99,999,999?  Odd that they didn't use contiguous memory for that.

Reply:
It was a while ago I did this one, I'm just catching & cleaning up and trying to get level with this branch.

GC: LFS 1,000,000,000
1st: LON 900,000,000
2nd: EDC 800,000,000
3rd: JEK 700,000,000
4th: JAK 600,000,000
5th: KRT 500,000,000

---Last Game---
51,563,560
3,671,330
90,510,050
46,782,010

![image](https://user-images.githubusercontent.com/5377332/205757705-2a33c55e-6b7b-4858-9213-981c85f797cc.png)

Reply:
Should KRT be 5th or 6th?  The DMD implies 6th.  I can't recall how Tommy does it's high scores.  Is GC just 1st?

Reply:
Yeah actually it should be 1st and not GC.